### PR TITLE
Newdesign

### PR DESCRIPTION
**Pull request by priyanka0901**

### CODE REVIEW

File: careers.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+  <title>Infratab Bangalore Pvt. Ltd</title>
```
Comment (priyanka1110) [inline careers.html:6]: @priyanka0901 Favicon icon should be added first if you are starting any project.

File: careers.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    </div>
+</section>
+
+<section class="join-us" id="js-join-us">
+    <p class="section-heading">Join us</p>
+    <p class="heading-bottom-line"></p>
+    <p class="section-subline">If you like getting things done, Infratab is the place for you. We are always looking for people who want to make a difference with their work.</p>
+    <div class="row">
+        <div class="col l3">
+            <div class="hiring-box">
```
Comment (priyanka1110) [inline careers.html]: @priyanka0901 Use materializescss card instead of writing your own structure and CSS

File: careers.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                <p class="hiring-require">Skills : Python, DJango, Postgresql, DRF</p>
+                <p class="hiring-button">Apply</p>
+            </div>
+        </div>
+        <div class="col l3">
+            <div class="hiring-box">
+                <p class="hiring-heading">Data Engineer</p>
+                <p class="hiring-require">Experience : 2+ years</p>
+                <p class="hiring-require">Skills : Python, MySQL, Data Analytics, Django</p>
+                <p class="hiring-button">Apply</p>
```
Comment (priyanka1110) [inline careers.html]: @priyanka0901  It should be the `button` or `a` tag as it's clear that it is a button.

File: careers.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    </div>
+    <p class="join-us-cv">Looking for a different role?</br>
+    Drop us a message with your CV at <a href="contact.html" class="contact-link">here</a> and we will get back to you.</p>
+</section>
+
+<footer class="footer">
+  <img src="img/waves.png" class="wave-image">
+  <div class="footer-content">
+    <div class="row">
+      <div class="col l2 m2 s2">
```
Comment (priyanka1110) [inline careers.html]: @priyanka0901 Grid should always go from small screen to big screen not from big to small. 
`For Ex: <div class="col s2 m2 l2"></div>`
If the grid value is same for all the screen then we don't have mention all the screen it should only `col s2`.

**Review by priyanka1110 — CHANGES_REQUESTED** [review]

File: careers.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+        <img src="img/infratab-logo.png" class="footer-logo">
+      </div>
+      <div class="col l2 m2 s2">
+         <p class="footer-heading">Contact</p>
+         <a class="footer-link"><i class="fa fa-envelope"></i> : info@infratab.in</a>
+         <a class="footer-link"><i class="fa fa-phone""></i> : (+91) 80 4092 9250</a>
+         <a class="footer-link"><i class="fa fa-globe"></i> : www.infratab.com</a>
+      </div>
+    </div>
+    <p class="horizontal-line"></p>
```
Comment (priyanka1110) [inline careers.html]: @priyanka0901 Why `p` tag for adding a horizontal line. There is two other way to do it adding `<hr>` tag or else border-bottom from the content container or else border-top from copyright section.

File: careers.html [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+    <p class="footer-copyright"><i class="fa fa-copyright"></i>Infratab Bangalore Pvt. Ltd. 2002-2017</p>
+  </div>
+</footer>
+<!--  Scripts-->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
+<script src="https://code.jquery.com/ui/1.9.2/jquery-ui.js"></script>
+<script src="https://use.fontawesome.com/0d9d5c45b7.js"></script>
```
Comment (priyanka1110) [inline careers.html]: @priyanka0901 Don't load CDN links use it locally.

**Review by priyanka1110 — CHANGES_REQUESTED** [review]

File: careers.html [hunk 7]
```diff
@@ -31,7 +30,7 @@
   <div class="team-overlay"></div>
   <div class="team-center">
       <p class="team-caption">BUILD future with us</p>
-      <p class="team-button">Join us</p>
+      <a class="team-button" id="js-join-scroll">Join us</a>
```
Comment (priyanka1110) [inline careers.html]: @priyanka0901 We should use only `camelcase` for the `ID` naming convention not lower case with hyphens. @Swathi8 Can correct me if I am wrong.

File: sass/pages/_careers.scss [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+.team-caption {
+  color: $white-color;
+  font-size: $font-40;
+  letter-spacing: .4px;
+  text-transform: uppercase;
+  word-spacing: 5px;
+}
+
+.team-button {
```
Comment (priyanka1110) [inline sass/pages/_careers.scss]: @priyanka0901 Why are you writing your own CSS for the button if it already available in the library.

File: careers.html [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+    </ul>
+    <a href="#" data-activates="nav-mobile" class="button-collapse"><i class="material-icons">MENUs</i></a>
+  </div>
+</nav>
+<section class="team scroll">
+  <div class="team-image"></div>
+  <div class="team-overlay"></div>
+  <div class="team-center">
+      <p class="team-caption">BUILD future with us</p>
+      <a class="team-button" id="js-join-scroll">Join us</a>
```
Comment (priyanka1110) [inline careers.html]: @priyanka0901 From the design you know that this raised button then it should be written in `button` tag, not `a` tag. `a` tag should be used only for linking and if you want to make the flat button.

File: careers.html (hunk 1 above)
Comment (priyanka0901) [reply careers.html:6]: I asked you told its fine to add font awsome

Comment (priyanka1110) [reply careers.html:6]: @priyanka0901 I am telling about favicon icon. https://en.wikipedia.org/wiki/Favicon

Comment (priyanka0901) [reply careers.html:6]: done

File: careers.html (hunk 2 above)
Comment (priyanka0901) [reply careers.html]: done

File: careers.html (hunk 3 above)
Comment (priyanka0901) [reply careers.html]: done

File: careers.html (hunk 5 above)
Comment (priyanka0901) [reply careers.html]: done

File: sass/pages/_careers.scss (hunk 8 above)
Comment (priyanka0901) [reply sass/pages/_careers.scss]: done

File: careers.html (hunk 9 above)
Comment (priyanka0901) [reply careers.html]: done

File: careers.html (hunk 6 above)
Comment (priyanka0901) [reply careers.html]: done

File: careers.html [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+    <p class="section-subline">Come, Watson, come!' he cried. 'The game is afoot. Not a word! Into your clothes and come!'</p>
+    <div class="row">
+        <div class="col s12 m6 l3">
+        <div class="card darken-1 ">
+            <div class="card-content">
+              <span class="card-heading">UI Developer</span>
+              <p>1 Positions</p>
+            </div>
+            <div class="card-action">
+              <a href="https://angel.co/infratab/jobs/45319-ui-developer" class="green-color">Apply</a>
```
Comment (priyanka1110) [inline careers.html]: @priyanka0901 Open `apply` button links into the new tab.

File: careers.html [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+            <p class="life-info-heading">Growth</p>
+            <p class="life-info">We care for each other and make them feel like they belong Encourage others to participate to their fullest. Listen, communicate openly and set clear expectations and ensure that they grow rapidly while working in joyful and optimism enviornment.</p>
+        </div>
+        <div class="col m6 l6">
+            <img src="img/teamgroup.png" class="group-image">
+        </div>
+    </div>
+    <div class="row margin-top-10">
+        <div class="col m6 l6 media-hide">
+            <img src="img/teamgroup.png" class="group-image">
```
Comment (priyanka1110) [inline careers.html:54]: @priyanka0901 Add proper folder for img.

**Comment by priyanka1110** [comment]
This PR is no longer required so closing it.

**Pull request closed without merging**